manim/renderer/opengl_renderer.py

```diff
@@ -386,30 +392,31 @@ def get_frame_buffer_object(self, context, samples=0):
         pixel_height = config["pixel_height"]
         num_channels = 4
         return context.framebuffer(
-            color_attachments=context.texture(
-                (pixel_width, pixel_height),
-                components=num_channels,
-                samples=samples,
-            ),
+            color_attachments=[
+                context.texture(
+                    (pixel_width, pixel_height),
+                    components=num_channels,
+                    samples=samples,
+                )
+            ],
             depth_attachment=context.depth_renderbuffer(
                 (pixel_width, pixel_height), samples=samples
             ),
         )
 
     def get_raw_frame_buffer_object_data(self, dtype="f1"):
-        # Copy blocks from the fbo_msaa to the drawn fbo using Blit
-        # pw, ph = self.get_pixel_shape()
-        # gl.glBindFramebuffer(gl.GL_READ_FRAMEBUFFER, self.fbo_msaa.glo)
-        # gl.glBindFramebuffer(gl.GL_DRAW_FRAMEBUFFER, self.fbo.glo)
-        # gl.glBlitFramebuffer(
-        #     0, 0, pw, ph, 0, 0, pw, ph, gl.GL_COLOR_BUFFER_BIT, gl.GL_LINEAR
-        # )
+        if hasattr(self, "frame_buffer_object_msaa"):
+            self.context.copy_framebuffer(
+                dst=self.frame_buffer_object, src=self.frame_buffer_object_msaa
+            )
+
```

Comment:
Why copy the data from the antialiased frame buffer to the regular rather than just return it from the antialiased frame buffer directly?